shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java

```diff
@@ -219,4 +161,8 @@ private String extractBrokerPath(KnativeBroker broker) {
             return null;
         }
     }
+
+    private UpdateControl<ManagedBridge> handleFailure(ManagedBridge managedBridge) {
+        return UpdateControl.updateStatus(managedBridge).rescheduleAfter(reconcileIntervalMillis);
```

Comment:
If you're stating (and we've all accepted) "time-out" is to be handled in the `manager` you can remove this method and just use the regular `UpdateControl.updateStatus(...)` - without the `reschedule(..)`. We added that specific method invocation when the "time-out" was handled in the `operator` to ensure our reconcile loop ran at determined intervals. Otherwise, if we had our loop invoked by k8s when events triggered it our "time-out" could have expired long ago.

Reply:
@robpblake @danielezonca @r00ta 
Are you all agree to remove `reconcileIntervalMillis` from `handleFailure`.

Reply:
Let's leave it in place for this PR to reduce the number of changes and then discuss https://issues.redhat.com/browse/MGDOBR-1308. We can remove it in a follow up PR pending what we decide.

Reply:
OK @robpblake.

In which case @vaibhavjainwiz you need to reinstate the time-out code in the [`Controller`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/90de10229b8a09280db7a0a4724a97dda6cf3e0c/shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java), unless it is elsewhere?

Reply:
@manstis Jira for same already exists.
https://issues.redhat.com/browse/MGDOBR-1308

Reply:
My point is; if you're not having time-out in the `operator` (and this PR removes it) then you don't need the use of `rescheduleAfter(..)` and this method along with the property can be removed. Time-out, the property and use of `rescheduleAfter(..)` all go hand-in-hand.

Reply:
got your point @manstis. Lets incorporate this changes in https://issues.redhat.com/browse/MGDOBR-1308